.github/workflows/release.yml

```diff
@@ -0,0 +1,42 @@
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tagname:
+        description: Release Tag Name
+        required: true
+        type: string
+
+env:
+  ACEVER: 0.9.34
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    env:
+      ACEFILE: "ace-${{ env.ACEVER }}-x86-64.tar.gz"
+      OUTFILE: "erg-${{ inputs.tagname }}-x86-64-$ACEVER.dat"
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.tagname }}
+
+      - name: Install ACE
+        run: |
+          wget "http://sweaglesw.org/linguistics/ace/download/$ACEFILE"
```

Comment:
Can we get the files from https://github.com/delph-in/ace? Maybe we can make a release there and use it until better automation in the Ace deploy too.

Reply:
What is the benefit of using the GitHub mirror over Woodley's server?

Reply:
1. a more stable link to download. We had problems in the past with Woodley's server being down
2. a push for DELPH-IN adoption of Git/GitHub
3. some automation on subsequent releases of Ace (via GitHub actions)
4. an easier experience for a newcomer showing better integration of the DELPH-IN ecosystem

Reply:
Thanks for explaining. Some responses:

1. Some of those problems were user error, like using `https` instead of `http`. His server has been fairly reliable in my experience, and I'm not too worried about having to re-run a job because the server happened to be down.
2. Maybe, but I don't think many people would notice this down in a CI script. And I'd say that sweaglesw.org is still the "official" home of ACE, so I don't know if it makes sense to promote the GitHub repo unless that changes.
3. I _would_ like to see an action to compile ACE on its GitHub repo, but we would still need to update this script to take advantage of future versions.
4. Not sure I see how using the GitHub ACE over the sweaglesw ACE achieves the easier experience for a newcomer.

Reply:
@goodmami, I expected to see the Action that you added in the list of Actions on https://github.com/delph-in/erg, but don't yet, so I guess it will appear once you commit your addition to main, yes?  The script looks like it will indeed do what I hope for, thanks.  Your call to compile the grammar is the same as what I do.  I don't think you should have to do anything to retrieve the redwoods.mem file, since I marked it to always be included for download.  (I know this costs more in Github data transfer than it would to store that file in some other repository somewhere, but if that starts to get expensive, we can revisit the setup.)

As for ACE, I think it remains best to get it from the place where Woodley maintains it, namely sweagles.  If he ever gets won over to using Github, we can adjust, but I agree that sweagles has been pretty reliable over time.

Reply:
@danflick thanks for the feedback. And, yes, the action should appear once this is merged.

The lack of visibility until it's merged makes testing a bit difficult. I was able to use https://github.com/nektos/act/ to test out most steps of this action locally, but I haven't tested the final step that uploads it as a release asset, so we may need to fix something after merging.

Reply:
I didn't know about this [nektosact.com](https://nektosact.com/), thank you for sharing!